transports/quic/src/endpoint.rs

```diff
@@ -280,6 +295,8 @@ pub enum ToEndpoint {
     Dial {
         /// UDP address to connect to.
         addr: SocketAddr,
+        /// Whether to dial the remote on QUIC version draft-29.
+        is_draft_29: bool,
```

Comment:
Instead of using a `bool`, I would prefer a new enum, e.g.:

``` rust
enum QuicVersion {
  V1,
  Draft29,
}
```

What do you think? Not a strong opinion.